ci/tasks/build_and_deploy/deploy.sh

```diff
@@ -16,5 +16,9 @@ aws sts assume-role --output text \
 export AWS_ACCESS_KEY_ID="$(cat AccessKeyId)"
 export AWS_SECRET_ACCESS_KEY="$(cat SecretAccessKey)"
 export AWS_SESSION_TOKEN="$(cat SessionToken)"
+export TF_DIST_ID=$(sed -n "2p" ../GITHUB_OUTPUT/output.txt | cut -c 15-)
+
 aws s3 sync ./build s3://sml-portal-$TF_VAR_environment-$workspace_name --delete --content-type "text/html" --exclude "*.css" --exclude "*.js"
 aws s3 sync ./build s3://sml-portal-$TF_VAR_environment-$workspace_name --delete --exclude "*" --include "*.css" --include "*.js"
+
+aws cloudfront create-invalidation --distribution-id $TF_DIST_ID --paths "/*"
```

Comment:
On line 3 could you update the comment to mention we invalidate caching to ensure the latest changes apply?

Reply:
done